news/2022-09-27-changes-to-osu-taiko-sr-and-pp.md

```diff
@@ -0,0 +1,153 @@
+
+In addition, a penalty was applied to repeat instances of these patterns, similar to how stamina was calculated. This only considered up to five recent occurrences, but if any two or more of such repeats occurred, a harsh penalty was applied.
+
+#### The new colour system
+
+To summarize shortly, the new colour system is more aware of *all* kinds of patterning used in osu!taiko mapping and is far more capable of formulating appropriate difficulty values from them compared to the old system.
+
+In general, the harder it is to describe a pattern, the harder the pattern will generally be considered. Additionally, the more dense a pattern (or color sequence, as described by the code) is, the harder the algorithm will consider it to be. Patterns that repeat with longer segments between them are also considered more difficult than patterns that repeat closely to one another.
+
+The code and concepts utilized to achieve this are quite extensive. If this sort of thing interests you, feel free to read more in the rework document listed below.
+
+### Calculating the final result
+
+Given the three skills briefly described above (stamina, rhythm and colour), their *strain* values have now become the number that approximates star rating. 
+
+Switching from the previous system, where strain and individual skill values formed star rating, strain is now the only summative factor in this calculation via the new *peak* skill.
+
+*Peak* difficulty splits the map into sections and combines the highest strains of each skill in each section, aiming to catch out particularly demanding moments. The final star rating is a weighted sum of the peak skill sections, including a slight multiplier to scale better.
+
+While this is all much more mathematically complex than described here, you can consult the [osu!taiko rework document](https://docs.google.com/document/d/1Z5GC4DMqOVzeIERMSK3qpQaqjq-sVnhbuoxAwy9qxDs/edit), or dive into the code over in the [osu!(lazer) GitHub repository](https://github.com/ppy/osu) for the full details.
+
+Previous approaches aimed to scale maps in regards to the [difficulty naming hierarchy](/wiki/Ranking_Criteria/Difficulty_naming), while this rework has deviated away from that goal, aiming instead to reflect true difficulty regardless of naming practices. 
+
+This also means that star rating values for maps with near-FCs are no longer near 9.5 stars, but rather can expand well into the 10 star range (medal hunters rejoice!).
+
+## Changes to performance points
+
+In recent years, osu!taiko has lagged behind the other three game modes in terms of raw performance point values, to the tune of being roughly 40% lower overall. In addition, the survey indicated a great preference for re-evaluating the way performance points are calculated.
```

Comment:
"lagged behind"? you mean the scale has been off? or time wise? development wise? confuse.

Reply:
the increase of overall top-end pp values for osu!taiko has been behind that of other gamemodes, is what it means

Reply:
Maybe a better way to phrase it is "on a different scale" or something? Is the goal here to bring them roughly into scale with expectations of osu! mode scores?